pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -2111,7 +2114,7 @@ void Object_endTypeContext(JSOBJ Py_UNUSED(obj), JSONTypeContext *tc) {
         NpyArr_freeLabels(GET_TC(tc)->columnLabels,
                           GET_TC(tc)->columnLabelsLen);
         GET_TC(tc)->columnLabels = NULL;
-
+        if (tc->type == JT_BIGNUM) free(GET_TC(tc)->bigNum_bytes);
```

Comment:
@WillAyd I know this isn't right (as is `ujson.encode` core dumps) but is it the right idea? If yes what should I look to fix

Reply:
Just clear cStr - you don't need bigNum_bytes